cmd/checkpath/checkpath_windows.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package checkpath
+
+import (
+	"path/filepath"
+)
+
+func pathsOnSameDevice(p1, p2 string) bool {
+	id1 := filepath.VolumeName(p1)
+	id2 := filepath.VolumeName(p2)
+	return id1 == id2
```

Comment:
Let's just shorten to `return filepath.VolumeName(p1) == filepath.VolumeName(p2)`

Reply:
Done.